tests/features/typing_pep563_test.py

```diff
@@ -0,0 +1,175 @@
+import sys
+
+import pytest
+
+from pyupgrade._data import Settings
+from pyupgrade._main import _fix_plugins
+
+
+@pytest.mark.parametrize(
+    ('s', 'version'),
+    (
+        pytest.param(
+            'from typing import Literal\n'
+            'x: "str"\n',
+            (2, 7),
+            id='not python 3+',
+        ),
```

Comment:
I think your test name here should be "not python3.10+" instead -- that's what you're actually testing in this case